src/gatling/simulations/uk/gov/hmcts/reform/cmc/performance/utils/Environment.scala

```diff
@@ -2,7 +2,7 @@ package uk.gov.hmcts.reform.cmc.performance.utils
 
 object Environment {
   val httpConfig = scala.util.Properties.envOrElse("httpConfig", "http")
-  
+
  val cmcBashURL = scala.util.Properties.envOrElse("baseURL", "https://cmc-citizen-frontend-perftest.service.core-compute-perftest.internal")
   val cmcLegalURL = scala.util.Properties.envOrElse("cmcLegalURL", "https://moneyclaims-legal.perftest.platform.hmcts.net")
  //val cmcBashURL = scala.util.Properties.envOrElse("baseURL", "https://cmc-citizen-frontend-aat.service.core-compute-aat.internal")
```

Comment:
i thought we are running performance tests in aat for nightly runs

Reply:
Again, this was how I found it, but I've wired the AAT env URLs in the `Jenkinsfile_nightly` so that we can overwrite them from there. Which will leave the default as running on the dedicated performance environment, which makes sense to me.

Reply:
And so only nightly runs in AAT.